causal_testing/specification/causal_dag.py

```diff
@@ -6,10 +6,17 @@
 
 Node = Union[str, int]  # Node type hint: A node is a string or an int
 logger = logging.getLogger(__name__)
+from .scenario import Scenario
+from .variable import Output
 
 
-def list_all_min_sep(graph: nx.Graph, treatment_node: Node, outcome_node: Node, treatment_node_set: {Node},
-                     outcome_node_set: {Node}):
+def list_all_min_sep(
```

Comment:
@jmafoster1 Why are the methods formatter like this? I have always preferred the former style -- open to changing though.

Reply:
It's just how Black does it. I have no strong preference either way. It's not something I've conciously changed.

Reply:
I would personally prefer not to use this kind of convention, it requires a lot of lines and doesn't make the code any more readable. I prefer to use the line width. Would you mind reverting these style changes?

Reply:
That'll mean I'll have to disable Black, which might make my code more anarchic in the long run. If you're OK with that then sure, although it might take a while to go through and revert all the methods that have been Blacked.